web/docs/advanced/accessing-app-config.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Accessing the configuration 
+---
+
+Whenever you start a Waps app, you are starting two processes.
+ - **The client process** - A React app that implements your app's frontend.
+
+    During development, this is a dev server with hot reloading. In production,
+    it's a Node process that serves pre-built static files with environment variables
+    embedded during the build.
+
+ - **The server process** - An Express server that implements your app's backend.
+ 
+  During development, this is an Express server controlled by a
+  [`nodemon`](https://www.npmjs.com/package/nodemon) process that takes care of
+  hot reloading and restarts. In production, it's a regular Express server run
+  using Node.
+
+Check [the introduction](/docs) for a more in-depth explanation of Wasp's runtime architecture.
+
+You can configure both processes through environment variables. See [the
+deployment instructions](/docs/advanced/deployment/manually#environment-variables) for a full list
+of supported variables.
+
+Wasp gives you runtime access to the processes' configurations through **configuration objects**.
+
+## Server configuration object
+
+The server configuration object contains these fields:
+
+- `frontendUrl: String` - Set it with env var `WASP_WEB_CLIENT_URL`.
+
+  The URL of your client (the app's frontend).<br/>
+  Wasp automatically sets it during development when you run `wasp start`.<br/>
+  In production, you should set it to your client's URL as the server sees it
+  (i.e., with the DNS and proxies considered).
+
+You can access it like this:
+```javascript
```

Comment:
Might I suggest using `js` instead of `javascript` for the language of the code block since we used it everywhere. It might help with future ripgreps :)

Reply:
I can change it, sure, but we definitely shouldn't rely on that when grepping (unless we make a build rule that enforces it).

Btw, I just grepped it, and there are already exceptions to this rule :smile: